File: Models/Comics.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Collections.Generic;

namespace Key_Comic_DB_Capstone.Models
{
    public class Comics
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public int IssueNumber { get; set; }
        public string CoverArtist { get; set; }
        public string StoryWriter { get; set; }
        public string CoverImage { get; set; }
        public string ComicReleased { get; set; }
    }
}

File: client/src/components/Comics.js
import React from "react";
import { Card, CardImg, CardBody } from "reactstrap";

const Comics = ({ comic }) => {
    return (
        <Card className="m-4" style={{ font: "", background: "lightgray", color: "", display: 'flex', flexDirection: 'column' }}>
            <CardBody>
                <p className="text-left px-2">Title: {comic.title}</p>
                <p className="text-left px-2">Issue: #{comic.issueNumber}</p>
                <p className="text-left px-2">Artist: {comic.coverArtist}</p>
                <p className="text-left px-2">Writer: {comic.storyWriter}</p>
                <p className="text-left px-2">Release Date: {comic.comicReleased}</p>
                <CardImg width="200px" height="300px" top src={comic.coverImage} alt={comic.title} />
            </CardBody>
        </Card>
    );
};

export default Comics;
